app/assets/stylesheets/projects.scss

```diff
@@ -106,6 +106,7 @@
 
 .projects-details-description {
```

Comment:
Add a CSS property
```css
width: 100%; 
```

For small message, it's taking small width instead of complete available width.
Except that it's working as expected

Reply:
Oh, okay sir on it